examples/quickstarts/cakephp-mysql.json

```diff
@@ -314,76 +324,102 @@
   ],
   "parameters": [
     {
+      "name": "MEMORY_LIMIT",
+      "displayName": "Memory limit",
+      "description": "Maximum amount of memory the CakePHP container can use",
+      "value": "128Mi"
+    },
+    {
+      "name": "MEMORY_MYSQL_LIMIT",
+      "displayName": "Memory limit",
+      "description": "Maximum amount of memory the MySQL container can use",
+      "value": "512Mi"
+    },
+    {
       "name": "SOURCE_REPOSITORY_URL",
+      "displayName": "Source repository URL",
       "description": "The URL of the repository with your application source code",
       "value": "https://github.com/openshift/cakephp-ex.git"
     },
     {
       "name": "SOURCE_REPOSITORY_REF",
+      "displayName": "Source repository reference",
       "description": "Set this to a branch name, tag or other ref of your repository if you are not using the default branch"
     },
     {
       "name": "CONTEXT_DIR",
+      "displayName": "Context directory",
       "description": "Set this to the relative path to your project if it is not in the root of your repository"
     },
     {
       "name": "APPLICATION_DOMAIN",
+      "displayName": "Application domain",
       "description": "The exposed hostname that will route to the CakePHP service, if left blank a value will be defaulted.",
       "value": ""
     },
     {
       "name": "GITHUB_WEBHOOK_SECRET",
+      "displayName": "GitHub webhook secret",
       "description": "A secret string used to configure the GitHub webhook",
       "generate": "expression",
       "from": "[a-zA-Z0-9]{40}"
     },
     {
       "name": "DATABASE_SERVICE_NAME",
+      "displayName": "Database service name",
       "description": "Database service name",
       "value": "mysql"
     },
     {
       "name": "DATABASE_ENGINE",
+      "displayName": "Database engine",
       "description": "Database engine: postgresql, mysql or sqlite (default)",
       "value": "mysql"
     },
     {
       "name": "DATABASE_NAME",
+      "displayName": "Database name",
```

Comment:
also redundant display name and description looks silly, drop the description if there is nothing more detailed to say